mapcreator/building.py

```diff
@@ -118,22 +127,15 @@ def get_output_path(input_path, new_extension = ''):
     new_filename += ('.' + new_extension if new_extension else '')
     return path.join(BUILD_DIR, new_filename)
 
-def check_projection_window(heightMapStatus, debug = False):
-    if heightMapStatus.state.has_window():
-        for cf in heightMapStatus.current_files:
-            gdal_info = gdal_util.Gdalinfo.for_file(cf)
-            cut_projection_window = heightMapStatus.state.get_window_string_lowerleft_topright_cut(gdal_info)
-            if cut_projection_window:
-                heightMapStatus.add_next_file(cf)
-    heightMapStatus.next()
-
 # Outputs only one combined file
 def process_heightfiles_with_gdal(heightMapStatus, debug = False):
     if heightMapStatus.current_files:    
         outpath = path.join(BUILD_DIR, INTERMEDIATE_HEIGHT_FILENAME)
         heightfiles = ' '.join(heightMapStatus.current_files)
         projection_window = heightMapStatus.state.get_window_string_lowerleft_topright()
         command = 'gdalwarp -tr {} {} -te_srs {} -t_srs {} -r bilinear -te {} {} {}'.format(OUTPUT_CELLSIZE, OUTPUT_CELLSIZE, LATLON_DATUM_IDENTIFIER, PROJECTION_IDENTIFIER, projection_window, heightfiles, outpath)
+        if heightMapStatus.state.has_height_system():
+            command = '{} -s_srs {} {}'.format(command[:8], heightMapStatus.state.height_coordinatesystem, command[9:])
```

Comment:
You should probably do this in the actual command formatting instead of here (string slicing like this is pretty dirty and not too easy to maintain)
Something like
```python
command = 'gdalwarp{} -tr {} {} -te_srs {} -t_srs {} -r bilinear -te {} {} {}'.format((' -s_srs' + heightMapStatus.state.height_coordinatesystem) if heightMapStatus.state.has_height_system() else '', OUTPUT_CELLSIZE, OUTPUT_CELLSIZE, LATLON_DATUM_IDENTIFIER, PROJECTION_IDENTIFIER, projection_window, heightfiles, outpath)
```
We could even refactor this formatting to be a little more descriptive and cleaner by using keyword arguments (see https://docs.python.org/3.4/library/string.html#format-string-syntax)